docs/notebooks/bsync_examples/Primary-School-Levels1-2.md

```diff
@@ -0,0 +1,8821 @@
+          </TypicalOccupantUsage>
+          <TypicalOccupantUsage>
+            <TypicalOccupantUsageValue>50.000000</TypicalOccupantUsageValue>
+            <TypicalOccupantUsageUnits>Weeks per year</TypicalOccupantUsageUnits>
+          </TypicalOccupantUsage>
+        </TypicalOccupantUsages>
+        <FloorAreas>
+          <FloorArea>
+            <FloorAreaType>Gross</FloorAreaType>
+            <FloorAreaValue>1840.620000</FloorAreaValue>
+          </FloorArea>
+          <FloorArea>
+            <FloorAreaType>Conditioned</FloorAreaType>
+            <FloorAreaValue>1840.620000</FloorAreaValue>
+          </FloorArea>
+        </FloorAreas>
+      </Section>
+      <Section ID="Whole-Building">
+        <SectionType>Whole building</SectionType>
+        <OccupancyClassification>Education-Primary</OccupancyClassification>
+        <OccupancyLevels>
+          <OccupancyLevel>
+            <OccupantQuantityType>Peak total occupants</OccupantQuantityType>
+            <OccupantQuantity>1849.000000</OccupantQuantity>
+          </OccupancyLevel>
+        </OccupancyLevels>
+        <TypicalOccupantUsages>
+          <TypicalOccupantUsage>
+            <TypicalOccupantUsageValue>45.000000</TypicalOccupantUsageValue>
+            <TypicalOccupantUsageUnits>Hours per week</TypicalOccupantUsageUnits>
+          </TypicalOccupantUsage>
+          <TypicalOccupantUsage>
+            <TypicalOccupantUsageValue>50.000000</TypicalOccupantUsageValue>
+            <TypicalOccupantUsageUnits>Weeks per year</TypicalOccupantUsageUnits>
+          </TypicalOccupantUsage>
+        </TypicalOccupantUsages>
+        <FloorAreas>
+          <FloorArea>
+            <FloorAreaType>Gross</FloorAreaType>
+            <FloorAreaValue>73960.000000</FloorAreaValue>
+          </FloorArea>
+          <FloorArea>
+            <FloorAreaType>Conditioned</FloorAreaType>
+            <FloorAreaValue>73960.000000</FloorAreaValue>
+          </FloorArea>
+        </FloorAreas>
+      </Section>
+    </Sections>
+    
+
+
+### 5.3 Plugs, lighting and HVAC
+
+Level 1 and Level 2 energy audits require information about the primary systems serving a specific section, but with very different degrees of specificity. Taking guidance from the 211 Normative spreadsheet:
+- A Level 1 audit just requires high level information
+- A Level 2 audit requires doing some more detailed modeling of the actual system of interest (mainly for HVAC).
+
+
+
+```python
+systems = bsync.Systems()
+# Systems, measures, reports and contacts belong to our facility (f1)
+f1 += systems
+
+# Let us automate the task this time
+# We first create a list of dictionaries that contains information on space IDs and the appropriate plug load. The plug load is taken from the Primary School Prototype ASHRAE Standard 90.1-2016 model
+# We take the IDs from the sections we created earlier
+plugloads = [{'spaceid' : lobby_sec["ID"], 'plugload' : 4.0},
```

Comment:
good job on the space functions! I'm wondering if in the future we can create a dict or something that we store all the space function data in first, then just iterate? Similar to your space plugloads here.

Reply:
Yes, agreed. I'll also do the same for schedules, it's a bit too dense right now IMHO.